runtime/vm/jvminit.c

```diff
@@ -6366,21 +6366,26 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 			 *
 			 * OpenJ9 stores both of these values as UDATA, so they will both be printed
 			 * as "size_t" type.
+			 *
+			 * NOTE that Hotspot produces this output on STDOUT, and applications expect to parse it there,
+			 * which means this ....
+			 * code does not use j9tty_printf like most of the VM (which prints to
+			 * STDERR). Instead, j9file_printf(PORTLIB, J9PORT_TTY_OUT, ...) is used.
```

Comment:
Thanks Mark.  I agree it doesn't make sense to take the principled stand here. 

Sorry to make another request, but can you reflow the comment so the lines are consistent lengths?  I was lazy in how I posted the suggested change and used `....` (which should be removed) rather than reflowing the full comment.
```suggestion
			 *
			 * NOTE that Hotspot produces this output on STDOUT, and applications
			 * expect to parse it there, which means this code does not use j9tty_printf
			 * like most of the VM (which prints to STDERR). Instead, 
			 * j9file_printf(PORTLIB, J9PORT_TTY_OUT, ...) is used.
```

Reply:
I'll fix it and then squash. Thanks for the suggestion; sorry for not paying close enough attention to your suggested change.